packages/identity/src/userSecret.ts

```diff
@@ -15,35 +17,22 @@ function checksumByte(secretRand: Uint8Array, userIdBytes: Uint8Array): number {
 }
 
 export function createUserSecretBinary(trustchainId: b64string, userId: string): Uint8Array {
-  if (typeof trustchainId !== 'string')
-    throw new Error('Assertion error: invalid Trustchain ID.');
-
-  if (typeof userId !== 'string')
-    throw new Error('Assertion error: invalid user ID.');
-
+  if (typeof trustchainId !== 'string') throw new Error('Assertion error: invalid Trustchain ID.');
+  if (typeof userId !== 'string') throw new Error('Assertion error: invalid user ID.');
```

Comment:
Since this is an internal function, are these type assertions necessary?

Reply:
these functions are exposed outside of the package. We only use these functions in our tests though. I'm not sure we would gain anything by removing the checks